compiler.rst

```diff
@@ -7,23 +7,14 @@ Design of CPython's Compiler
 Abstract
 --------
 
-Historically (through 2.4), compilation from source code to bytecode
-involved two steps:
+In CPython, the compilation from source code to bytecode involves three steps:
```

Comment:
I was confused by that too TBH. From what I understand, Brett (who originally wrote this as I understand) considered the last two as a single step because, as stated later:

> The conversion process is initiated by a call to the function PyAST_Compile() in Python/compile.c . *This function does both the conversion of the AST to a CFG and outputting final bytecode from the CFG*.